cypress/page/login.page.ts

```diff
@@ -0,0 +1,26 @@
+class LoginPage {
+    private clickLogin: string;
+    private emailInput: string
+    private passInput: string
+
+    constructor() {
+        this.emailInput = "#email"
+        this.passInput = "#passwd"
+        this.clickLogin = "#SubmitLogin";
+
+    }
+
+    public insertMail(email: string): void {
+        cy.get(this.emailInput).type(email)
+    }
+
+    public insertPassword(pass: string): void {
+        cy.get(this.passInput).type(pass)
+    }
+
+    public goSubmit(): void {
```

Comment:
Since we are login into the app, we can call this method as login to provide more context about the action we are performing

Reply:
Changed to login